packages/react-server-examples/bike-share/package.json

```diff
@@ -7,35 +7,21 @@
   "author": "Doug Wade <doug.wade@redfin.com>",
   "scripts": {
     "clean": "rm -rf build __clientTemp",
-    "start": "npm run clean && npm run styles && react-server start",
-    "styles": "node-sass styles/index.scss build/styles/index.css && node-sass styles/network.scss build/styles/network.css",
-    "test": "xo && nsp check"
+    "start": "npm run clean && react-server start --hot",
+    "test": "nsp check"
   },
   "license": "Apache-2.0",
   "dependencies": {
-    "babel-runtime": "^6.6.1",
+    "babel-runtime": "^6.11.6",
```

Comment:
This _shouldn't_ even be necessary?

Should only need the packages that are installed by [`react-server init`](https://github.com/redfin/react-server/blob/master/packages/react-server-cli/src/commands/init.js#L8-L16).